box2d/README.md

```diff
@@ -271,6 +272,19 @@ val body = world.body {
 }
 ```
 
+Creating ray-casts:
+
+```Kotlin
+import ktx.box2d.*
+
+fun createRayCast() {
+  world.rayCast(0f, 0f, 1f, 1f) { fixture, point, normal, fraction ->
```

Comment:
Could you include named parameters in the `rayCast` invocation to make it a bit clearer? 4 subsequent floats are not something I'd consider an obvious API. ; )